tools/system_libs.py

```diff
@@ -615,6 +615,42 @@ def get_default_variation(cls, **kwargs):
     return super().get_default_variation(eh_mode=eh_mode, **kwargs)
 
 
+class SjLjLibrary(Library):
```

Comment:
How about calling this `LongjmpLibrary`?  Would that better describe its purpose?

Reply:
I don't have strong opinions on names, but the functionality handles both setjmp and longjmp, so why would say only longjmp? But yeah we already have the setting name as `SUPPORT_LONGJMP`, so it might be better to be consistent, but all LLVM options use `sjlj`. What's your reason for using 'longjmp' in Emscripten side?